Makefile

```diff
@@ -82,8 +117,17 @@ test100:
 vagrant_test:
 	vagrant up
 	vagrant ssh -c 'make install'
-	vagrant ssh -c 'make test_race'
-	vagrant ssh -c 'make test_integrations'
+	vagrant ssh -c 'make test_unit_race'
+	vagrant ssh -c 'make test_all'
+
+### go tests without docker
+test_unit:
```

Comment:
I feel like we should still keep `test` target. Maybe alias it to `test_unit` by default?

```
test: test_unit
```

Reply:
let's just keep `test` to conform with: https://github.com/tendermint/tendermint/blob/381fe19335ba8825e04d1d0fefa5cac709bb7178/test/test_libs.sh#L15